autorag/evaluate/metric/generation.py

```diff
@@ -124,29 +125,43 @@ async def compute(gt: List[str], pred: str) -> float:
     return result
 
 
-@generation_metric
-def sem_score(generation_gt: List[str], pred: str, embedding_model: Optional[BaseEmbedding] = None) -> float:
+def sem_score(generation_gt: List[List[str]], generations: List[str],
+              embedding_model: Optional[BaseEmbedding] = None) -> List[float]:
```

Comment:
??? Where is the batch input...?
My review was "change default setup of batch", not delete as input parameter.